nautobot/extras/api/nested_serializers.py

```diff
@@ -4,9 +4,9 @@
 
 from nautobot.core.api import BaseModelSerializer, ChoiceField, ContentTypeField, WritableNestedSerializer
 from nautobot.core.api.exceptions import SerializerNotFound
+from nautobot.core.utils.api import get_serializer_for_model
```

Comment:
Good example here - I'd strongly prefer this to be `nautobot.core.api.utils` rather than `nautobot.core.utils.api`